### PR TITLE
Fixing autoquant after inductor moved do_bench

### PR DESCRIPTION
**Pull request by HDCharles**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #156

Summary: this was moved in pytorch.

Test Plan: python test/integration/test_integration.py -k "autoquant"

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

File: torchao/quantization/autoquant.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     safe_int_mm,
 )
 import torch.nn.functional as F
-from torch._inductor.utils import do_bench
+from torch._inductor.runtime.runtime_utils import do_bench
```
Comment (cpuhrsch) [inline torchao/quantization/autoquant.py:13]: probably needs a version if/else

**Comment by cpuhrsch** [comment]
Changed the base to main and merged upstream to trigger CI

**Comment by cpuhrsch** [comment]
Test currently fails because import needs a version guard

**Comment by HDCharles** [comment]
yeah that makes sense

**Comment by cpuhrsch** [comment]
This seems to fail on nightly now?

**Comment by msaroufim** [comment]
Fixed upstream already

**Pull request closed without merging**